pl_examples/basic_examples/README.md

```diff
@@ -2,62 +2,70 @@
 
 Use these examples to test how Lightning works.
 
-#### MNIST
+## MNIST Examples
 
-Trains MNIST where the model is defined inside the `LightningModule`.
+Here are 5 MNIST examples showing you how to gradually convert from pure PyTorch to PyTorch Lightning.
+
+The transition through [LightningLite](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.rst) from pure PyTorch is optional but it might be helpful to learn about it.
+
+#### 1. Image Classifier with Vanilla PyTorch
+
+Trains a simple CNN over MNIST using vanilla PyTorch.
 
 ```bash
 # cpu
-python simple_image_classifier.py
+python image_classifier_1_pytorch.py
+```
+
+______________________________________________________________________
 
-# gpus (any number)
-python simple_image_classifier.py --trainer.gpus 2
+#### 2. Image Classifier with LightningLite
 
-# Distributed Data Parallel
-python simple_image_classifier.py --trainer.gpus 2 --trainer.accelerator ddp
+This script shows you how to scale the previous script to enable GPU and multi-GPU training using [LightningLite](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html).
+
+```bash
+# CPU / multiple GPUs if available
+python image_classifier_2_lite.py
 ```
 
 ______________________________________________________________________
 
-#### MNIST with DALI
+#### 3. Image Classifier - Conversion from Lite to Lightning
```

Comment:
```suggestion
#### 3. Image Classifier - Introducing LightningModule powered by Lite
```

Reply:
> Introducing LightningModule powered by Lite

But that's not what it is. There is no LightningModule in that step